Sources/Colors/BrandsSampleView/BlocketColorsView.swift

```diff
@@ -2,7 +2,7 @@ import SwiftUI
 
 public struct BlocketColorsView: View {
     public init() {}
-
+    
```

Comment:
There are some white spaces in the following lines that seem unnecessary. These changes aren't needed, right?

Reply:
I used `cmd+A` to select all and then `ctrl+I` to reindent to whatever `Xcode` decided is the right indentation 😅
So probably was wrong before